types/pm-rpc.d.ts

```diff
@@ -0,0 +1,42 @@
+export interface TargetDefinition {
+  target?: MessagePort | Worker | Window | HTMLIFrameElement;
+}
+
+export declare type OnApiCall = (data: {
+  appId: string;
+  call: string;
+  args: any[];
+}) => void;
```

Comment:
```suggestion
export declare type OnApiCall<T extends unknown[]> = (data: {
  appId: string;
  call: string;
  args: T;
}) => void;
```

?

Reply:
Replaced to this. and use OnApiCall<unknown[]> in set interfacce